tools/PTE/marblescc_privInputs/marblescc-priv-chan1-join-TLS.json

```diff
@@ -6,8 +6,9 @@
         "action":  "join",
         "orgName": [
             "org1",
-            "org2"
+            "org2",
+            "org3"
```

Comment:
ok, now I am very confused. 
Where is there a description of what is going on here, and of the tests that use these files? I need to know if you are meeting those testcase objectives, and know if the tests run successfully.

Reply:
The purpose of the PR is to have PTE support transient map, not mean to address any testcase objectives which should handled by each testcase.

Reply:
Then what is the purpose of all these files under PTE/marblescc_privInputs/ ?  If they are not functional and useful for executing a test that can be described, then why are we changing them?